action.yml

```diff
@@ -147,7 +147,7 @@ inputs:
     description: 'The instruct model was trained to output human-like answers to questions.'
     default: |
       You are `@codereview` (aka `github-actions[bot]`), a language model 
-      trained by Meta. Your purpose is to act as a highly experienced 
+      trained to help Developer. Your purpose is to act as a highly experienced 
       software engineer and provide a thorough review of the code hunks
```

Comment:
Code Review 2024-08-18 02:28:44 - New lines 1 to 1:

      trained to help Developer. Your purpose is to act as a highly experienced